shell/platform/darwin/macos/framework/Source/FlutterTextInputPlugin.mm

```diff
@@ -351,7 +466,7 @@ - (void)insertText:(id)string replacementRange:(NSRange)range {
   if (range.location != NSNotFound) {
     // The selected range can actually have negative numbers, since it can start
     // at the end of the range if the user selected the text going backwards.
-    // NSRange uses NSUIntegers, however, so we have to cast them to know if the
+    // NSRange uses NSUIntegers, however, so they have to be casted to know if the
```

Comment:
This wording seems more awkard than the original. Do we have something in the style guide about avoiding the word `we` in implementation comments? I'd have assumed that only applies to user-facing public doc comments that are likely to be viewed outside the context of the code itself.

What about this?
```
// Cast to a signed type to determine whether or not the selection is reversed.
```